webpack.prod.js

```diff
@@ -3,7 +3,6 @@ const { merge } = require('webpack-merge')
 const TerserPlugin = require('terser-webpack-plugin')
 
 module.exports = merge(common, {
-  devtool: 'eval',
```

Comment:
maybe change to `devtool: 'source-map'` instead, see https://webpack.js.org/configuration/devtool/

Reply:
Did this now